### ISSUE TITLE
Unable to install add-on HASS 2022.9.4

### ISSUE THREAD

**Issue by Nurovico** [sdesalve/hassio-addons#56]
After installing Sdesalve repository in HASS add-on section the DSS VoIP do not appear at all, so I was unable to install it.

I follow all instructions but after install sdesalve url into HASS add-on Repositories option the add-on do not appear in the list.

I'm using:
Home Assistant 2022.9.4
Supervisor 2022.08.6
Operating System 9.0
Frontend 20220907.2 - latest

on:

Raspberry Pi 4 [rpi4-64]

<img width="763" alt="Screen Shot 2022-09-18 at 7 05 07 PM" src="https://user-images.githubusercontent.com/108946747/190919567-6c31ed0a-4877-4428-8464-de1309abffdf.png">

**Comment by sdesalve** [comment]
I'm your addon will not be visible DSS VoIP Notifier for ARM?

**Comment by Nurovico** [comment]
There are not DSS VoIP. In HASS website said: "If you have added a repository, but it’s not showing up, it means that the repository contains invalid configuration".

Is there a problem with the new HASS version?

O provato con altro add-on e sucede lo steso, 
Non c'e niente dopo la installazione del link.

**Comment by sdesalve** [comment]
Seems might be some problems with your hassio. Have you tried to restart host machine and please post your hassio logs after adding addon repository

**Comment by Nurovico** [comment]
I restart several times and I got the same result

**Comment by sdesalve** [comment]
Logs?

Otherwise I must tell you that addon works for every other people and I can't help

**Comment by Nurovico** [comment]
Finally I get the add-on, after installing your url several times and delete it, in my last reboot (the one I used to send you the logs) it finally works.

I installed the DSS VoIP on my HASS 

Now I 'll try to make it work with my provider {Netelip}

I'll let you up to date.

One last question: is is possible to make regular calls? I mean a regular cell or land line? all examples are based on call_sip_uri":"sip:+393334455667@sipserver.com for example.

Graze infinite per il tuo aiuto!
